core/modules/system/system.install

```diff
@@ -2505,7 +2505,34 @@ function system_update_1026() {
   $config->set('file_temporary_path', update_variable_get('file_temporary_path', ''));
   $config->set('file_private_path', update_variable_get('file_private_path', ''));
   $config->set('image_jpeg_quality', update_variable_get('image_jpeg_quality', 75));
-  $config->set('install_profile', update_variable_get('install_profile', 'standard'));
+
+  // Get the Drupal install profile and switch it to "standard" if not usable
+  // with Backdrop.
+  $current_profile = update_variable_get('install_profile', 'standard');
+  if ($current_profile != 'standard') {
+    $core_profiles = file_scan_directory('./core/profiles', '/\.profile$/', array('key' => 'name'));
+    $root_profiles = file_scan_directory('./profiles', '/\.profile$/', array('key' => 'name'));
+    $available = array_merge($core_profiles, $root_profiles);
+    if ($current_profile == 'minimal' || !array_key_exists($current_profile, $available)) {
```

Comment:
I read the comments in the issue about the minimal profile, but I don't agree with switching it here to avoid possible future problems *if* it's removed. I think we should remove it from this PR as it's currently a valid profile in Backdrop.

```suggestion
    if (!array_key_exists($current_profile, $available)) {
```